studio/schemas/schemaTypes/slug.ts

```diff
@@ -19,26 +19,44 @@ async function isSlugUniqueAcrossAllDocuments(
   return result;
 }
 
+const SLUG_MAX_LENGTH = 200;
+
 function createSlugField(source: string) {
   return defineField({
     type: "slug",
     name: "slug",
     title: "URL Path (slug)",
     description:
-      "Enter a unique URL path for the page. This path will be used in the website's address bar. A URL path, also known as a slug, is a URL-friendly version of the page title, used to create a human-readable and search engine optimized URL for the content.",
+      "Enter a unique URL path for the page. This path will be used in the website's address bar. A URL path, also known as a slug, is a URL-friendly version of the page title, used to create a human-readable and search engine optimized URL for the content. Legal characters include latin letters, digits, hyphen (-), underscore (_), full stop (.) and tilde (~)",
     options: {
       source,
-      maxLength: 200,
+      maxLength: SLUG_MAX_LENGTH,
       slugify: (input) =>
-        input.toLowerCase().replace(/\s+/g, "-").slice(0, 200),
+        input
+          .toLowerCase()
+          // replace æøå according to https://sprakradet.no/spraksporsmal-og-svar/ae-o-og-a-i-internasjonal-sammenheng/
+          .replace(/[æ,å]/g, "a")
+          .replace(/ø/g, "o")
+          .replace(/[^a-zA-Z0-9-_.~\s]/g, "") // remove non-whitespace URL-unsafe chars (section 2.3 in https://www.ietf.org/rfc/rfc3986.txt)
+          .trim()
+          .replace(/\s+/g, "-")
+          .slice(0, SLUG_MAX_LENGTH),
       isUnique: isSlugUniqueAcrossAllDocuments,
     },
-    validation: (Rule) => Rule.required(),
+    validation: (Rule) =>
+      Rule.required().custom((value) => {
+        if (value?.current === undefined) return true;
+        return (
+          encodeURIComponent(value.current) === value.current ||
+          "Slug can only consist of latin letters, digits, hyphen (-), underscore (_), full stop (.) and tilde (~)"
+        );
+      }),
   });
 }
 
 const pageSlug = createSlugField("page");
-const blogSlug = createSlugField("basicTitle");
-const legalSlug = createSlugField("basicTitle");
+const titleSlug = createSlugField("basicTitle");
+const blogSlug = titleSlug;
+const legalSlug = titleSlug;
 
-export { pageSlug, blogSlug, legalSlug };
+export { pageSlug, titleSlug, blogSlug, legalSlug };
```

Comment:
```suggestion
export { pageSlug, pageTitleSlug };
```
then you can replace blogSlug, legalSlug with titleSlug